portable/MSVC-MingW/portmacro.h

```diff
@@ -129,9 +129,13 @@ void vPortExitCritical( void );
     #endif
 
 /* Store/clear the ready priorities in a bit map. */
+#if defined( __x86_64__ ) || defined( _M_X64 )
+    #define portRECORD_READY_PRIORITY( uxPriority, uxReadyPriorities )    ( uxReadyPriorities ) |= ( 1ULL << ( uxPriority ) )
+    #define portRESET_READY_PRIORITY( uxPriority, uxReadyPriorities )     ( uxReadyPriorities ) &= ~( 1ULL << ( uxPriority ) )
+#else
     #define portRECORD_READY_PRIORITY( uxPriority, uxReadyPriorities )    ( uxReadyPriorities ) |= ( 1UL << ( uxPriority ) )
     #define portRESET_READY_PRIORITY( uxPriority, uxReadyPriorities )     ( uxReadyPriorities ) &= ~( 1UL << ( uxPriority ) )
-
+#endif
 
```

Comment:
Can we avoid adding conditional compilation here by using `( ( UBaseType_t ) 1 )` instead of `1UL`?

Reply:
Done